style.css

```diff
@@ -90,6 +90,10 @@
     .introduction{
         margin-right: 106px;
```

Comment:
The year inside the button looks weird because it's not center.

![Screenshot 2023-03-13 at 2 24 12 PM](https://user-images.githubusercontent.com/38748298/224657322-69f0c0f6-b078-4262-b92d-50a35cd20a46.png)

Reply:
I have fixed this, this was due to the height property.